rotcon/convolve.py

```diff
@@ -41,12 +41,18 @@ def __init__(self, *args, **kwargs):
               the labelled sites
 
         :Keywords:
-           *dcdFilename*
+           *dcdFilenameAll*
```

Comment:
change back to `dcdFilename`, there's no real need to change the name and it breaks the API